docs/source/usersguide/materials.rst

```diff
@@ -0,0 +1,169 @@
+.. _usersguide_materials:
+
+.. currentmodule:: openmc
+
+=====================
+Material Compositions
+=====================
+
+Materials in OpenMC are defined as a set of nuclides/elements at specified
+densities and are created using the :class:`openmc.Material` class. Once a
+material has been instantiated, nuclides can be added with
+:meth:`Material.add_nuclide` and elements can be added with
+:meth:`Material.add_element`. Densities can be specified using atom fractions or
+weight fractions. For example, to create a material and add Gd152 at 0.5 atom
+percent, you'd run::
+
+   mat = openmc.Material()
+   mat.add_nuclide('Gd152', 0.5, 'ao')
+
+The third argument to :meth:`Material.add_nuclide` can also be 'wo' for weight
+percent. The densities specified for each nuclide/element are relative and are
+renormalized based on the total density of the material. The total density is
+set using the :meth:`Material.set_density` method. The density can be specified
+in gram per cubic centimeter, atom per barn-cm, or kilogram per cubic meter,
+e.g.,
+
+::
+
+   mat.set_density('g/cm3', 4.5)
+
+----------------
+Natural Elements
+----------------
+
+The :meth:`Material.add_element` method works exactly the same as
+:meth:`Material.add_nuclide`, except that instead of specifying a single isotope
+of an element, you specify the element itself. For example,
+
+::
+
+   mat.add_element('C', 1.0)
+
+Internally, OpenMC stores data on the atomic masses and natural abundances of
+all known isotopes and then uses this data to determine what isotopes should be
+added to the material. When the material is later exported to XML for use by the
+:ref:`scripts_openmc` executable, you'll see that any natural elements are
+expanded to the naturally-occurring isotopes.
+
+Often, cross section libraries don't actually have all naturally-occurring
+isotopes for a given element. For example, in ENDF/B-VII.1, cross section
+evaluations are given for O16 and O17 but not for O18. If OpenMC is aware of
+what cross sections you will be using (either through the
+:attr:`Materials.cross_sections` attribute or the
```

Comment:
Ditto, this doesn't show up as a hyperlink, though perhaps it isn't possible to designate it as such with Sphinx.